mulighetsrommet-api/src/main/resources/application-prod.yaml

```diff
@@ -75,3 +75,8 @@ app:
 
   norg2:
     baseUrl: https://norg2.prod-fss-pub.nais.io/norg2/api/v1
+
+  slack:
+    token: ${SLACK_PROD_TOKEN}
```

Comment:
Akkurat denne secret'en kunne evt. vært `SLACK_TOKEN` i alle miljøer, da Secret Manager "instans" varierer for dev og prod, men har kanskje ikke så mye å si

Reply:
Godt poeng. Jeg fikser så det blir likt